src/dipdup/config.py

```diff
@@ -968,6 +970,11 @@ class AdvancedConfig:
     metadata_interface: bool = False
 
 
+class CustomConfig(BaseModel):
+    class Config:
+        extra = Extra.allow.value
```

Comment:
Are there any advantages of the plain Pydantic model over the regular `Dict[str, Any]'?